greenmail-core/src/main/java/com/icegreen/greenmail/imap/commands/FetchCommand.java

```diff
@@ -37,6 +48,7 @@
     public static final String ARGS = "<message-set> <fetch-profile>";
     private static final Flags FLAGS_SEEN = new Flags(Flags.Flag.SEEN);
     private static final Pattern NUMBER_MATCHER = Pattern.compile("^\\d+$");
+    private static final SimpleDateFormat INTERNALDATE_FORMAT = new SimpleDateFormat("dd-MMM-yyyy HH:mm:ss Z");

```

Comment:
I agree in principle, but I don't think there is any hard requirement on this and I have seen some responses specifying non-zero offsets, so its seems prudent to include this (and the client should know how to handle it regardless)